solr/core/src/java/org/apache/solr/search/function/FileFloatSource.java

```diff
@@ -307,7 +307,8 @@ private static float[] getFloats(FileFloatSource ffs, IndexReader reader) {
         } catch (Exception e) {
           if (++otherErrors <= 10) {
             log.error(
-                "Error loading external value source + fileName + {}{}",
+                "Error loading external value source:{}{}{}",
```

Comment:
This concats the messages without whitespace. Can you correct this?

Reply:
> This concats the messages without whitespace. Can you correct this?

sure!

Reply:
I mean behind the file name. The last two `{}{}` are fine, because they 2nd one is optional and adds whitespace inside the appendix.

This one is still missing whitespace between filename and exception.